cg/meta/orders/case_submitter.py

```diff
@@ -213,59 +212,79 @@ def order_to_status(order: OrderIn) -> dict:
         return status_data
 
     def store_items_in_status(
-        self, customer: str, order: str, ordered: dt.datetime, ticket: str, items: List[dict]
-    ) -> List[models.Family]:
-        """Store cases and samples in the status database."""
-
-        customer_obj = self.status.customer(customer)
-        new_families = []
+        self, customer_id: str, order: str, ordered: dt.datetime, ticket_id: str, items: List[dict]
+    ) -> List[Family]:
+        """Store cases, samples and their relationship in the Status database."""
+        customer: Customer = self.status.get_customer_by_customer_id(customer_id=customer_id)
+        new_cases: List[Family] = []
         for case in items:
-            case_obj = self.status.family(case["internal_id"])
-            if not case_obj:
-                case_obj = self._create_case(case, customer_obj, ticket)
-                new_families.append(case_obj)
+            status_db_case: Family
+            existing_case: Family = self.status.family(case["internal_id"])
```

Comment:
Can we not just use one variable here?
```suggestion
            status_db_case: Family = self.status.family(case["internal_id"])
```

Reply:
I think you are correct - fixed